pandas-stubs/io/formats/style.pyi

```diff
@@ -1,97 +1,333 @@
+        self,
+        subset: Subset | None = ...,
+        axis: Axis = ...,
+        level: Level | list[Level] | None = ...,
+        names: bool = ...,
+    ) -> Styler: ...
     def background_gradient(
         self,
-        cmap=...,
-        low=...,
-        high=...,
-        axis=...,
-        subset=...,
-        text_color_threshold=...,
+        cmap: str | Colormap = ...,
+        low: float = ...,
+        high: float = ...,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        text_color_threshold: float = ...,
+        vmin: float | None = ...,
+        vmax: float | None = ...,
+        gmap: Sequence[float]
+        | Sequence[Sequence[float]]
+        | npt.NDArray
+        | DataFrame
+        | Series
+        | None = ...,
+    ) -> Styler: ...
+    def text_gradient(
+        self,
+        cmap: str | Colormap = ...,
+        low: float = ...,
+        high: float = ...,
+        axis: Axis | None = ...,
+        subset: Subset | None = ...,
+        # In docs but not in function declaration
+        # text_color_threshold: float
         vmin: float | None = ...,
         vmax: float | None = ...,
-    ): ...
-    def set_properties(self, subset=..., **kwargs): ...
+        gmap: Sequence[float]
+        | Sequence[Sequence[float]]
+        | npt.NDArray
+        | DataFrame
+        | Series
+        | None = ...,
+    ) -> Styler: ...
+    def set_properties(
+        self, subset: Subset | None = ..., **kwargs: str | int
+    ) -> Styler: ...
     def bar(
         self,
-        subset=...,
-        axis: int = ...,
+        subset: Subset | None = ...,
+        axis: Axis | None = ...,
+        *,
+        color: str | list[str] | tuple[str, str] | None = ...,
+        cmap: str | Colormap = ...,
+        width: float = ...,
+        height: float = ...,
+        align: Literal["left", "right", "zero", "mid", "mean"]
+        | float
+        | Callable[[Series | npt.NDArray | DataFrame], float] = ...,
+        vmin: float | None = ...,
+        vmax: float | None = ...,
+        props: str = ...,
+    ) -> Styler: ...
+    def highlight_null(
+        self,
+        null_color: str = ...,
+        subset: Subset | None = ...,
+        props: str | None = ...,
+    ) -> Styler: ...
+    def highlight_max(
+        self,
+        subset: Subset | None = ...,
+        color: str = ...,
+        axis: Axis | None = ...,
+        props: str | None = ...,
+    ) -> Styler: ...
+    def highlight_min(
+        self,
+        subset: Subset | None = ...,
+        color: str = ...,
+        axis: Axis | None = ...,
+        props: str | None = ...,
+    ) -> Styler: ...
+    def highlight_between(
+        self,
+        subset: Subset | None = ...,
+        color: str = ...,
+        axis: Axis | None = ...,
+        left: Scalar | list[Scalar] | None = ...,
+        right: Scalar | list[Scalar] | None = ...,
+        inclusive: Literal["both", "neither", "left", "right"] = ...,
```

Comment:
Can be `IntervalClosedType` from `_typing.pyi`

Reply:
Fixed.